### PR TITLE
Update Device Role Descriptions

### PR DESCRIPTION
**Pull request by tekstrand**
## What
Updates device descriptions to match @garthvh's changes in the various client applications

## Why
The current descriptions left some ambiguity causing frustration and misconfiguration by users

## Resources
Google Doc - This version is likely easier to read than the markdown here in GitHub. You may also leave comments in the Google sheet to suggest changes.

## Screenshots
Screenshot 2024-02-07 at 7 32 19 PM

| Device Role    | Description                                                                                                               | Best Uses                                                                                                                              |
| -------------- | ------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------- |
| CLIENT         | App connected or stand alone messaging device.                                                                            | General use for individuals needing to communicate over the Meshtastic network with support for client applications.                   |
| CLIENT_MUTE    | Device that does not forward packets from other devices.                                                                  | Situations where a device needs to participate in the network without assisting in packet routing, reducing network load.              |
| CLIENT_HIDDEN  | Device that only broadcasts as needed for stealth or power savings.                                                       | Use in stealth/hidden deployments or to reduce airtime/power consumption while still participating in the network.                     |
| TRACKER        | Broadcasts GPS position packets as priority.                                                                              | Tracking the location of individuals or assets, especially in scenarios where timely and efficient location updates are critical.      |
| LOST_AND_FOUND | Broadcasts location as message to default channel regularly for to assist with device recovery.                           | Used for recovery efforts of a lost device.                                                                                            |
| SENSOR         | Prioritizes broadcasting GPS position packets.                                                                            | Deploying in scenarios where gathering environmental or other sensor data is crucial, with efficient power usage and frequent updates. |
| TAK            | Optimized for ATAK system communication, reduces routine broadcasts.                                                      | Integration with ATAK systems for communication in tactical or coordinated operations.                                                 |
| REPEATER       | Infrastructure node for extending network coverage by relaying messages with minimal overhead. Not visible in Nodes list. | Best positioned in strategic locations to maximize the network's overall coverage. Device is not shown in topology.                    |
| ROUTER         | Infrastructure node for extending network coverage by relaying messages. Visible in Nodes list.                           | Best positioned in strategic locations to maximize the network's overall coverage. Device is shown in topology.                        |
| ROUTER_CLIENT  | Combination of both ROUTER and CLIENT. Not for mobile devices.                                                            | Devices in a strategic position for priority routing that need to also serve as a standard CLIENT.                                     |

### CODE REVIEW

**Comment by garthvh** [comment]
I updated the English strings in the iOS app, modified them a bit 

https://github.com/meshtastic/Meshtastic-Apple/blob/ef59160b163347f7d66625b29a3a2146c4df8756/en.lproj/Localizable.strings#L70

They are also in TestFlight under the pick list for role once you have selected a role.

**Comment by tekstrand** [comment]
While this is ready for review, it is **not** ready for merge until changes can be released to the various applications.

**Comment by mtsdawson** [comment]
Re: the TAK role - this outlines **dedicated** node use but doesn't address **shared** use between ATAK and Meshtastic.  Many users will likely want to use both apps concurrently for testing.  It might help to specify what, if any, features are disabled in the Meshtastic app when the role is configured to TAK.  This would reduce confusion by guiding the user to leave the node in CLIENT role when planning to use both apps.

**Comment by tekstrand** [comment]
@mtsdawson given this requires more context, what are your thoughts about providing a link to ATAK specific documentation rather than giving the context here where most users will not need it?

**Comment by mtsdawson** [comment]
That might be a better fix, if the ATAK documentation covers **shared use** between both apps.  I think the majority of users will be sharing (vs true, pure ATAK-only node deployments)... so the negative impact of this role should be surfaced along with the feasibility of leaving the node in CLIENT role when using ATAK.

File: docs/configuration/radio/device.mdx [hunk 1]
```diff
@@ -11,24 +11,19 @@ The device config options are: Role, Serial Output, and Debug Log. Device config
...
+| -------------- | ------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------- |
+| CLIENT         | App connected or stand alone messaging device.                                                                            | General use for individuals needing to communicate over the Meshtastic network with support for client applications.                   |
+| CLIENT_MUTE    | Device that does not forward packets from other devices.                                                                  | Situations where a device needs to participate in the network without assisting in packet routing, reducing network load.              |
+| CLIENT_HIDDEN  | Device that only broadcasts as needed for stealth or power savings.                                                       | Use in clandestine operations or to reduce airtime/power consumption while still participating in the network.                         |
```
[the remark this answers is not in the document]
Comment (tekstrand) [reply docs/configuration/radio/device.mdx:19]: Agree. @garthvh you good with that?

**Comment by mtsdawson** [comment]
That might be a better fix, if the ATAK documentation covers shared use between both apps. I think the majority of users will be sharing (vs true, pure ATAK-only node deployments)... so the negative impact of this role should be surfaced along with the feasibility of leaving the node in CLIENT role when using ATAK.

File: docs/configuration/radio/device.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (tekstrand) [reply docs/configuration/radio/device.mdx:19]: Must've been a bad copy/paste on my part. Will get it fixed later.

**Comment by tekstrand** [comment]
♻️ with `client_hidden` suggestions. Just waiting on @garthvh to give the thumbs up in relation to the strings in the apps.

**Comment by garthvh** [comment]
Looks good to me I will get the app updated to match exactly

**Review by garthvh — APPROVED** [review]
Deployed in the App Store now as well

**Pull request merged**